paths/invitations/update.yaml

```diff
@@ -34,14 +38,14 @@ x-code-samples:
     curl "https://api.phrase.com/v2/accounts/:account_id/invitations/:id" \
       -u USERNAME_OR_ACCESS_TOKEN \
       -X PATCH \
-      -d '{"role":"Invitiation role","project_ids":"abcd1234abcd1234abcd1234,abcd1234abcd1234abcd1235","locale_ids":"abcd1234abcd1234abcd1234,abcd1234abcd1234abcd1235","default_locale_codes":["de","en"],"permissions":{"create_upload":true}}' \
+      -d '{"role":"Invitiation role","project_ids":"abcd1234abcd1234abcd1234,abcd1234abcd1234abcd1235","locale_ids":"abcd1234abcd1234abcd1234,abcd1234abcd1234abcd1235","space_ids":["abcd1234abcd1234abcd1234", "default_locale_codes":["de","en"], "abcd1234abcd1234abcd1235"],"permissions":{"create_upload":true}}' \
```

Comment:
I think something about the ordering of the arguments is not correct. Now the `default_locale_codes` are within the `space_ids`